repose-aggregator/components/filters/filter-bundle/pom.xml

```diff
@@ -204,6 +204,11 @@
             <artifactId>cors</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.openrepose</groupId>
+            <artifactId>ip-classification</artifactId>
```

Comment:
that is the question. i like the idea of these types of filters having a common name, but i'm not 100% sold on -user. I understand why we went with it, but i'm open to better names.